### ISSUE TITLE
Example in Guide/Best Practices/Accessibility#Placeholder is missing

### ISSUE THREAD

**Issue by c0lt** [vuejs/docs#1532]
I just read the new guide and I think the example in the [Placeholder](https://vuejs.org/guide/best-practices/accessibility.html#placeholder) section of the accessibility page is missing. The text says:
`Looking at the following example, you can see that the Last Name placeholder which meets the color contrast criteria looks like pre-populated data:`, but there is nothing to be seen.

**Comment by skirtles-code** [comment]
It seems that all the CodePens in that file have been commented out and not replaced with anything.

**Comment by skirtles-code** [comment]
This is partially fixed by #1564.

I think a more complete fix would be to convert all the CodePen examples to Playgrounds.

**Comment by c0lt** [comment]
I think there is no need for the code example at this place but for an image illustrating the problem. In the CodePen example, the placeholder of the last name field is styled differently and that''s the point to be shown (adjusting color contrast of the placeholder makes it look like normal input data). Here''s the output of the CodePen:

<img width="95" alt="image" src="https://user-images.githubusercontent.com/9382537/154943060-5aaf59c6-d0e6-4bb8-9977-ef5b8bc2e6a7.png">

And the different styling is not included in the new code template.
